### ISSUE TITLE
WebView does not work

### ISSUE THREAD

**Issue by SULAPIS** [sciter-sdk/rust-sciter#139]
When I run the program, WebView does not display the web page. The examples of this repositories can run successfully in my computer and I also use the sciter-webview.dll file. Did I do anything wrong？These are my codes.

webview.html
```html
<html>

<head>
    <title></title>
    <style>
        html,
        body {
            padding: 0;
            margin: 0;
        }

        webview {
            display: block;
            size: 1000;
            background: red;
            behavior: webview library(sciter-webview);
        }
    </style>

</head>

<body>

    <webview src="https://www.sciter.com"></webview>
</body>

</html>
```
main.rs
```rust
use sciter;

fn main() {
    let mut frame = sciter::Window::new();
    frame.load_file("webview.html");
    frame.run_app();
}
```
<pre>
<- rust                                                             scapp.exe->
</pre>
![image](https://user-images.githubusercontent.com/81067069/185764913-d363fb5d-8871-4e25-accb-1efaf3e2a05c.png)

**Comment by c-smile** [comment]
Do you have sciter-webview.dll in the same folder as sciter.dll  ?

**Comment by SULAPIS** [comment]
yes,they are in same folde

**Comment by SULAPIS** [comment]
> Do you have sciter-webview.dll in the same folder as sciter.dll ?

yes,they are in same folde

**Comment by SULAPIS** [comment]
> Do you have sciter-webview.dll in the same folder as sciter.dll ?

yes,they are in same folder

**Comment by AnassKartit** [comment]
it is not working with tiscript and version 4.4.9, is it supported only with js sdk and version 5.0?  @c-smile

**Comment by c-smile** [comment]
`library` part here:
```CSS
behavior: webview library(sciter-webview);
```
is supported by Sciter.JS only.

In principle you can use webview with Sciter.TIS but you will need to either compile that plugin statically with your app or create a local proxy behavior for that DLL.

**Comment by AnassKartit** [comment]
is it possible to use sciter js and tiscript at the same time, i am trying to add oauth to tiscript with a webview for authentication, in electron this is working with no issue https://learn.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-nodejs-desktop

**Comment by c-smile** [comment]
> is it possible to use sciter js and tiscript at the same time,

No. I'd suggest to port your TIS code to JS. Normally that's line-to-line porting.
